packages/react-charting/src/components/SankeyChart/SankeyChart.base.tsx

```diff
@@ -1,15 +1,15 @@
-import * as React from 'react';
-import { classNamesFunction, getId, getRTL, memoizeFunction } from '@fluentui/react/lib/Utilities';
-import { ISankeyChartData, ISankeyChartProps, ISankeyChartStyleProps, ISankeyChartStyles } from './SankeyChart.types';
+import { FocusZone, FocusZoneDirection, FocusZoneTabbableElements } from '@fluentui/react-focus';
+import { Callout, DirectionalHint } from '@fluentui/react/lib/Callout';
 import { IProcessedStyleSet } from '@fluentui/react/lib/Styling';
-import { SankeyLayout, SankeyGraph, sankey as d3Sankey, sankeyRight, sankeyJustify } from 'd3-sankey';
-import { area as d3Area, curveBumpX as d3CurveBasis } from 'd3-shape';
+import { classNamesFunction, getId, getRTL, memoizeFunction } from '@fluentui/react/lib/Utilities';
 import { sum as d3Sum } from 'd3-array';
-import { ChartHoverCard, IBasestate, IChartHoverCardProps, SLink, SNode } from '../../index';
-import { Callout, DirectionalHint } from '@fluentui/react/lib/Callout';
+import { SankeyGraph, SankeyLayout, sankey as d3Sankey, sankeyJustify, sankeyRight } from 'd3-sankey';
 import { select, selectAll } from 'd3-selection';
-import { FocusZone, FocusZoneDirection, FocusZoneTabbableElements } from '@fluentui/react-focus';
+import { area as d3Area, curveBumpX as d3CurveBasis } from 'd3-shape';
+import * as React from 'react';
+import { ChartHoverCard, IBasestate, IChartHoverCardProps, SLink, SNode } from '../../index';
 import { IMargins } from '../../utilities/utilities';
+import { ISankeyChartData, ISankeyChartProps, ISankeyChartStyleProps, ISankeyChartStyles } from './SankeyChart.types';
```

Comment:
We have another open task to reduce the overall bundle size for the charting package.
[https://github.com/microsoft/fluentui/pull/30244](https://github.com/microsoft/fluentui/pull/30244)

We are starting with obfuscating the tree chart and plan to do it for Sankey chart next.

Reply:
That sounds good. I am not trying to do absolute bundle size mitigation, here. I am just tackling some low-hanging fruit that I would normally fix anyway. I am doing this as a separate PR so that these changes do not cloud my next PR.